javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,19 @@
+function getEventWeekday(days) {
```

Comment:
nitty gritty (nit): indentation of the code is important for readability. consider using the format feature of vscode :)